components/add_new_cv_form.tsx

```diff
@@ -86,27 +93,88 @@ export default function AddNewCvForm({ id }: Props) {
     }
 
     delete updatedCv.projects;
-    delete updatedCv.technical_skills;
+    delete updatedCv.certifications;
+    delete updatedCv.education;
 
     return supabase.from("cv").upsert(updatedCv).select();
   }
 
-  async function upsertSkills(values: any, cvData: any) {
-    values.cv_id = cvData[0].id;
-    return supabase.from("technical_skills").upsert(values);
+  async function upsertCertifications(certifications: any, cvId: string) {
+    if (certifications.length === 0) {
+      return await supabase.from("certifications").delete().eq("cv_id", cvId);
+    }
+
+    const updatedCertifications = certifications.map((certification: any) => ({
+      ...certification,
+      cv_id: cvId,
+    }));
+    return supabase.from("certifications").upsert(updatedCertifications);
+  }
+
+  async function upsertEducation(education: any, cvId: string) {
+    education.cv_id = cvId;
+    return supabase.from("education").upsert(education);
+  }
+
+  async function upsertProjects(projects: any, cvId: string) {
+    if (projects.length === 0) {
+      return await supabase.from("projects").delete().eq("cv_id", cvId);
+    }
+
+    const updatedProjects = projects.map((project: any) => {
+      const startDate = new Date(project.date_start);
+      const endDate = new Date(project.date_end);
+      const formattedStartDate = `${startDate.getDay() + 1}.${
```

Comment:
Just curious, why we're increasing day value by 1? I know month value is 0-11, but days are 1-31.

Reply:
My bad, I think it my intension was to use `getDate` not `getDay`, but it doesn't really matter since we only care about month and year.